test/test-api-wmata.js

```diff
@@ -0,0 +1,41 @@
+require('dotenv').config();
+var path = require('path');
+var tape = require('tape-expect');
+var tapeNock = require('tape-nock');
+var wmataApi = require('../lib/api-wmata.js');
+var props = require('../resources/properties.json');
+
+var test = tapeNock(tape, {
+  fixtures: path.join(__dirname, 'fixtures')
+});
+
+test('wmata api get station prediction valid key', function (t) {
+  var url = props.stationArrivalWmataUrl + 'B02';
+  wmataApi.get(url, function (response) {
+    t.plan(2);
+    t.ok(response);
+    t.expect(response.Trains[0]).to.have.keys('Car', 'Destination', 'DestinationCode', 'DestinationName', 'Group',
+      'Line', 'LocationCode', 'LocationName', 'Min');
+    t.end();
+  });
+});
+
+test('wmata api get station prediction invalid key', function (t) {
```

Comment:
Since our code is a wrapper over the WMATA API there isn't much to test. Writing the tests was still useful - it uncovered the necessity for [this line](https://github.com/pmyers88/dc-metro-echo/blob/test-wmata/lib/api-wmata.js#L13) to return `res` when `error` doesn't exist. 

I still think this is useful as an integration test. I want to make it so that you can call `wmataApi.getStationPrediction` and `wmataApi.getAdvisories` so that the client isn't responsible of knowing the URL for those resources. With these tests I can make that change, enable nock to make real requests and see if it works right away.